zerver/openapi/zulip.yaml

```diff
@@ -11935,6 +11944,15 @@ paths:
                       to the current user. **Changes**: New in Zulip 8.0 (feature level 232). This
                       capability is for backwards-compatibility.
 
+                    - `archived_streams`: Boolean for whether the client's
+                      handler for the `stream` event type want to get results
+                      including archived streams. Otherwise, the server will
+                      send `stream` events only for non-archived streams.
```

Comment:
This should use the same writing style as the others.

"Boolean for whether the client supports processing archived streams in the `stream` and `subscription` event types. If `false`, the server will not include in the `register` response or in events related to archived streams."

Reply:
Fixed.